newm-server-portal/src/main/kotlin/io/projectnewm/server/portal/model/Song.kt

```diff
@@ -0,0 +1,19 @@
+package io.projectnewm.server.portal.model
+
+import kotlinx.datetime.LocalDate
+import kotlinx.datetime.serializers.LocalDateIso8601Serializer
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Song(
+    @SerialName("id") val id: Int,
```

Comment:
Should this be a UUID instead of an Int?

Reply:
Good call. I changed to a String and updated the mock data with UUIDs